rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/Row.java

```diff
@@ -94,4 +97,22 @@ public void add(Field field) {
         }
         return nonPkFields;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
equals and hashCode together?

Reply:
i forgot it, i will submit a new commit at the moment